src/test/cypress/support/pageobjects/lecture/LectureManagementPage.ts

```diff
@@ -3,59 +3,59 @@ import day from 'dayjs';
 
 export class LectureManagementPage {
     clickCreateLecture() {
-        cy.get('.create-lecture').click();
+        cy.get('#jh-create-entity').click();
     }
 
     deleteLecture(lectureTitle: string) {
-        this.getLectureRow(lectureTitle).find('.btn-danger').click();
-        cy.get('.modal-footer').find('.btn-danger').should('be.disabled');
-        cy.get('.modal-body').find('input').type(lectureTitle);
+        this.getLectureRow(lectureTitle).find('#delete-lecture').click();
+        cy.get('#delete').should('be.disabled');
+        cy.get('#confirm-exercise-name').type(lectureTitle);
         cy.intercept(DELETE, `${BASE_API}lectures/*`).as('deleteLecture');
-        cy.get('.modal-footer').find('.btn-danger').click();
+        cy.get('#delete').click();
         return cy.wait('@deleteLecture');
     }
 
     getLectureRow(lectureTitle: string) {
-        return this.getLectureSelector(lectureTitle).parents('tr');
+        return this.getLectureSelector(lectureTitle).parents();
```

Comment:
I think it is better to give the tablerow simply a unique id like we did it for the course and exercise cards. For example `lecture-{{ id }}`. This way we don't have to match the lectures via their title and can do it with one selector instead of 3 like it is currently.

Reply:
👍